src/main/resources/com/google/api/codegen/java/initcode.snip

```diff
@@ -30,6 +30,30 @@
   @end
 @end
 
+@private initCodeLinesWithDescriptions(lines)
```

Comment:
I suggest factoring out the `@switch` statement so that both `initCodeLines` and `initCodeLinesWithDescription` can call a single source of truth. Maybe `emitInitCodeLine` can be the function that holds the `@switch`?

Alternatively, we could have a single `initCodeLines(lines, printDescription)` with an extra arg. I don't like this quite as much because it's less informative at the call site.

This applies to all the languages.